chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs

```diff
@@ -0,0 +1,240 @@
+﻿using System.Data;
+using Microsoft.Data.Sqlite;
+using Microsoft.VisualBasic.FileIO;
+using System.Collections.Generic;
+using System.Globalization;
+
+
+class Program
+{
+	static string connectionString = @"Data Source=habit-Tracker.db";
+
+	static void Main(string[] args)
+	{
+		using (var connection = new SqliteConnection(connectionString))
+		{
+			connection.Open();
+			var tableCmd = connection.CreateCommand();
+
+			tableCmd.CommandText = @"CREATE TABLE IF NOT EXISTS drinking_water (
+								Id INTEGER PRIMARY KEY AUTOINCREMENT,
+								Date TEXT,
+								Quantity INTEGER
+								)";
+			tableCmd.ExecuteNonQuery();
+			connection.Close();
+		}
+
+		GetUserInput();
+
```

Comment:
Please remove unnecessary empty line 29. You want to have one line between methods but none between closing curly brackets